src/main/java/com/mparticle/kits/ApptentiveKit.java

```diff
@@ -40,11 +55,30 @@ public String getName() {
 		if (KitUtils.isEmpty(apptentiveAppSignature)) {
 			throw new IllegalArgumentException("Apptentive App Signature is required. If you are migrating from a previous version, you may need to enter the new Apptentive App Key and Signature on the mParticle website.");
 		}
-		ApptentiveConfiguration configuration = new ApptentiveConfiguration(apptentiveAppKey, apptentiveAppSignature);
+
+    ApptentiveConfiguration configuration = new ApptentiveConfiguration(apptentiveAppKey, apptentiveAppSignature);
 		Apptentive.register((Application)context.getApplicationContext(), configuration);
+
+		Activity activity = getActivity();
+		if (activity != null) {
+			ApptentiveActivityLifecycleCallbacks.getInstance().onActivityCreated(activity, null);
```

Comment:
this shouldn't be necessary - our KitManager replays these lifecycle events automatically.

Reply:
@samdozor, our SDK requires to be registered in the `Application.onCreate` method where it subscribes for the activity lifecycle callbacks. If we miss any of the the activity lifecycle's events - we're not gonna be able to detect background/foreground events correctly.

Reply:
@samdozor, let me run a few tests really quick to see if our SDK still works as intended.

Reply:
@samdozor, I've verified the behavior. Everything works fine after removing manual activity lifecycle calls. Thanks for pointing that out!